vars/infra.groovy

```diff
@@ -342,6 +342,14 @@ void prepareToPublishIncrementals() {
  */
 void maybePublishIncrementals() {
   if (new InfraConfig(env).isRunningOnJenkinsInfra() && currentBuild.currentResult == 'SUCCESS') {
+    if (env.CHANGE_ID == null) {
+      catchError(message: 'Could not check whether repo has enabled CD', buildResult: 'SUCCESS', stageResult: 'UNSTABLE', catchInterruptions: false) {
```

Comment:
In case `readTrusted` fails, for example because `.mvn/maven.config` does not exist (though in that case it would be impossible for Incrementals publishing to work anyway), or because it has been modified in an untrusted revision (though this should only happen when `CHANGE_ID` is set). Erring on the side of the status quo.